torch/quantization/make_modules.py

```diff
@@ -0,0 +1,100 @@
+import os
+
+_disclaimer = r"""
+# This file is generated using `torch.nn.quantization.make_module`
```

Comment:
This comment should not be present in this file

Reply:
Actually, I think it should be. We need to clearly mark the files that can be overwritten as generated. Otherwise, a de developer can just modify them, and next time we run the generation, all changes will be lost